src/fuselage/CCPACSPressureBulkheadAssemblyPosition.cpp

```diff
@@ -83,13 +82,6 @@ TopoDS_Shape CCPACSPressureBulkheadAssemblyPosition::GetGeometry()
     return m_geometry.value();
 }
 
-void CCPACSPressureBulkheadAssemblyPosition::ReadCPACS(const TixiDocumentHandle &tixiHandle, const std::string &xpath)
-{
-    generated::CPACSPressureBulkheadAssemblyPosition::ReadCPACS(tixiHandle, xpath);
-    
-    if (m_uidMgr && !m_uID.empty()) m_uidMgr->AddGeometricComponent(m_uID, this);
```

Comment:
NO!!! :smile:  I added it intentionally. The auto-generated class does not register it as a geometric component! Therefore, TiGL Viewer does not show this geometry in the list of possibilities. Please revert this or make sure, that all geometric components that are auto-registered are found as geometric component by the uid manager!

Reply:
But the bulkhead is shown here:
![image](https://user-images.githubusercontent.com/1224051/40651086-268cad7c-6335-11e8-9463-1692baed75a3.png)
Or should it be shown somewhere else?

Furthermore, AddGeometricComponent() is private on CTiglUIDManager for exactly the reason that registration should happen automatically.

Reply:
No... This looks perfect ! Last time, I tried this, it did not work! Sorry for the inconvenience.

Great refactoring! Hard for me, to track all the recent changes in my brain ;)